src/main/resources/config/liquibase/changelog/190821_1322_event_category.xml

```diff
@@ -21,7 +21,7 @@
                                  baseTableName="Event_Category"
                                  constraintName="fk_eventIdConstraint"
                                  referencedColumnNames="id"
-                                 referencedTableName="Event"/>
+                                 referencedTableName="event"/>
```

Comment:
De ce schimbi numele tabelelor din fisierele de config? Te rog sa te asiguri ca au probleme si ceilalti din echipa si merita aceasta modificare, altfel inseamna ca ai o problema pe statia ta.

Reply:
Pai exista diferente intre numeleTabelei atunci cand este declarata si numeleTabelei cand este referentiata in sensul ca nu mai tin cont daca incepea sau nu cu litera mare. Este o problema cu statia mea in sensul ca tine cont de diferentele astea, dar ma gandesc ca e mai buna varianta in care programul ruleaza fara probleme pentru toata lumea, respectiv cazul in care numele declarat si cel referentiat coincid.

Reply:
Daca modifici si le produci lor probleme, atunci nu este ok. Mai bine ramai tu cu probleme decat sa avem 10 cu aceasta problema.

Reply:
Ok